CSSLayout/CSSLayout.c

```diff
@@ -2549,7 +2549,7 @@ void CSSLog(CSSLogLevel level, const char *format, ...) {
   va_end(args);
 }
 
-static bool experimentalFeatures[CSSExperimentalFeatureCount + 1];
```

Comment:
Let's keep this in here as it is very little memory and changing this back and forth when adding/removing experiments is just annoying.